packages/template/vite-typescript/tmpl/forge.config.ts

```diff
@@ -3,7 +3,7 @@ import { MakerSquirrel } from '@electron-forge/maker-squirrel';
 import { MakerZIP } from '@electron-forge/maker-zip';
 import { MakerDeb } from '@electron-forge/maker-deb';
 import { MakerRpm } from '@electron-forge/maker-rpm';
-import { VitePlugin } from '@electron-forge/plugin-vite';
+import { VitePlugin } from 'electron-forge-plugin-vite';
```

Comment:
Since `@electron-forge/plugin-vite` is not yet released, we are using `electron-forge-plugin-vite` instead of passing CI, and will replace it with `@electron-forge/plugin-vite` once CI is passsed.

> `electron-forge-plugin-vite` is the same code as `@electron-forge/plugin-vite`